### DISCUSSION TITLE
Cannot connect to localhost:25565

### DISCUSSION THREAD

**Discussion by Peregrine05** [mircokroon/minecraft-world-downloader#259] [Q&A]
I am trying to download a server world. The server is running Minecraft 1.17. I have input the server IP and set a World output. The Server port and the Local port options are at default. Then I click 'Start'. But when I try to connect to localhost:25565 in Minecraft 1.17, the connection times out on "Encrypting..." What could be the problem?
I have tried other servers as well, but the outcome is the same.

**Comment by Peregrine05** [comment]
This could be a firewall issue. I am testing right now.

**Reply by WhynotFulp** [reply]
Hey! Did you ever figure out how to get past this issue? I am experiencing the same.

**Reply by Peregrine05** [reply]
Downloading an earlier version of the software seemed to fix it. Are you using Windows 7? I am using Windows 7. So, if you are also, then that might be the issue.

**Comment by WhynotFulp** [comment]
Hey!
I am using windows 10 or 11 so that's not the issue. Were you able to get a version of this mod working with 1.18? Can you share instructions on how to get it working? I am trying to download a realm map but so far I have run into problems that have to do with the mod telling me to connect to localhost and something about that not working.
-CM

**Reply by Peregrine05** [reply]
The software does not support Minecraft 1.18 yet. Does it get stuck on Encrypting? That is what happened to me. I tried several older versions of the software until I got one that worked. It might have been v1.14.3 or v1.14.4, both of which allow Microsoft accounts. But if you're on Windows 11, then you might need v1.14.9. However, it is the latest stable release, and it does not work for me.

**Reply by WhynotFulp** [reply]
![Screenshot 2022-02-18 224729](https://user-images.githubusercontent.com/99362800/155202252-0c143b4e-7d12-4c85-b451-11476885692d.png)
![Screenshot 2022-02-18 224946](https://user-images.githubusercontent.com/99362800/155202255-97ca7817-b8d5-40b5-ba33-4ea7e079727a.png)
![Screenshot 2022-02-18 225101](https://user-images.githubusercontent.com/99362800/155202256-897e9b62-f455-48e3-8775-b406279606a6.png)
![Screenshot 2022-02-18 225116](https://user-images.githubusercontent.com/99362800/155202264-dbe34924-cb58-4411-b882-20ecfa6e2fb0.png)
![Screenshot 2022-02-18 225140](https://user-images.githubusercontent.com/99362800/155202268-39f08b9f-247f-4f18-b78f-aa321571fd9b.png)
![Screenshot 2022-02-18 225818](https://user-images.githubusercontent.com/99362800/155202271-0fc84b51-960d-4b02-897d-22eac603f954.png)

**Comment by WhynotFulp** [comment]
I am playing 1.18 on windows 11 with an account that is not yet migrated, so you think I should try WDL version v1.14.3 or v1.14.4 or v1.14.9?

**Reply by Peregrine05** [reply]
With 1.18 worlds, you need the [v1.15 pre-release](https://github.com/mircokroon/minecraft-world-downloader/releases/tag/v1.15). That has limited support for 1.18 and 1.18.1 worlds. If you are using 1.17.1, then you need [v1.14.9](https://github.com/mircokroon/minecraft-world-downloader/releases/tag/v1.14.9), which supports authentication on Windows 11.

**Comment by StarWarsExperte** [comment]
I can not connect to the localhost too no matter which version of the programm I tried. I authentificated my account, put in the localhost adress. When I press start, the server's logo is shown in the localhost server. But when I join it, I get the error 
Failed to connect to the server
Internal Exception: java.net.SocketException: Connection reset

**Reply by froggygroggy** [reply]
did you ever find a way to fix this? i am having the same issue.
